ErgoLux/localization/strings.tr-TR.resx

```diff
@@ -0,0 +1,728 @@
+  <data name="strDlgReset" xml:space="preserve">
+    <value>Tüm alanları varsayılan değerlerine sıfırlamak ister misiniz?</value>
+  </data>
+  <data name="strDlgResetTitle" xml:space="preserve">
+    <value>Yenileme ayarlanıyor ?</value>
+  </data>
+  <data name="strErrorDeserialize" xml:space="preserve">
+    <value>Ayarlar dosyası yüklenirken hata oluştu.
+{0}
+
+Bunun yerine varsayılan değerler kullanılacaktır..</value>
+  </data>
+  <data name="strErrorDeserializeTitle" xml:space="preserve">
+    <value>Hata</value>
+  </data>
+  <data name="strFileHeader01" xml:space="preserve">
+    <value>ErgoLux verisi</value>
+  </data>
+  <data name="strFileHeader02" xml:space="preserve">
+    <value>Başlangıç zamanı</value>
+  </data>
+  <data name="strFileHeader03" xml:space="preserve">
+    <value>Bitiş zamanı</value>
+  </data>
+  <data name="strFileHeader04" xml:space="preserve">
+    <value>Toplam ölçüm süresi</value>
+  </data>
+  <data name="strFileHeader05" xml:space="preserve">
+    <value>Sensör Sayısı</value>
+  </data>
+  <data name="strFileHeader06" xml:space="preserve">
+    <value>Veri puan sayısıs</value>
+  </data>
+  <data name="strFileHeader07" xml:space="preserve">
+    <value>Örnekleme frekansı</value>
+  </data>
+  <data name="strFileHeader08" xml:space="preserve">
+    <value>Sensör #</value>
+  </data>
+  <data name="strFileHeader09" xml:space="preserve">
+    <value>En büyük</value>
+  </data>
+  <data name="strFileHeader10" xml:space="preserve">
+    <value>ortalama</value>
+  </data>
+  <data name="strFileHeader11" xml:space="preserve">
+    <value>En küçük</value>
+  </data>
+  <data name="strFileHeader12" xml:space="preserve">
+    <value>En küçük /Ortalama</value>
+  </data>
+  <data name="strFileHeader13" xml:space="preserve">
+    <value>En küçük / En büyük</value>
+  </data>
+  <data name="strFileHeader14" xml:space="preserve">
+    <value>Ortalama/ En büyük</value>
+  </data>
+  <data name="strFileHeader15" xml:space="preserve">
+    <value>Ortalama</value>
+  </data>
+  <data name="strFileHeader16" xml:space="preserve">
+    <value>Aydınlık</value>
+  </data>
+  <data name="strFileHeader17" xml:space="preserve">
+    <value>Eksik alanlar var</value>
+  </data>
+  <data name="strFileHeader18" xml:space="preserve">
+    <value>Eksik sütun başlıkları (seri adları ).</value>
+  </data>
+  <data name="strFileHeader19" xml:space="preserve">
+    <value>günler, gün, günler</value>
+  </data>
+  <data name="strFileHeader20" xml:space="preserve">
+    <value>saatler, saat, saatler</value>
+  </data>
+  <data name="strFileHeader21" xml:space="preserve">
+    <value>dakikalar, dakika, dakikalar</value>
+  </data>
+  <data name="strFileHeader22" xml:space="preserve">
+    <value>saniyeler, saniye, saniye</value>
+  </data>
+  <data name="strFileHeader23" xml:space="preserve">
+    <value>ve</value>
+  </data>
+  <data name="strFileHeader24" xml:space="preserve">
+    <value>mili saniyeler, mili saniye, mili saniyeler</value>
```

Comment:
All the above strings (`strFileHeader19` to `strFileHeader24`) should be treated as pointed out in the previous comment.